optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/tenant/TenantRestServiceImpl.java

```diff
@@ -55,14 +55,15 @@ public Tenant addTenant(Tenant tenant) {
 
     @Override
     @Transactional
-    public Tenant updateTenantConfiguration(TenantConfiguration tenantConfiguration) {
-        Tenant tenant = entityManager.find(Tenant.class, tenantConfiguration.getTenantId());
-        if (null == tenant) {
-            throw new IllegalStateException("There is no tenant with id (" +
-                                            tenantConfiguration.getTenantId() + ").");
-        }
-        tenant.setConfiguration(tenantConfiguration);
-        return entityManager.merge(tenant);
+    public TenantConfiguration updateTenantConfiguration(TenantConfiguration tenantConfiguration) {
+        return entityManager.merge(tenantConfiguration);
+    }
+
+    @Override
+    public TenantConfiguration getTenantConfiguration(Integer tenantId) {
```

Comment:
method order, to make it easier to read/maintain and see if it's consistenty:
get before add, add before update, update before delete.